### DISCUSSION TITLE
How to fix 'subfinder' not found error

### DISCUSSION THREAD

**Discussion by bakliwalp** [projectdiscovery/subfinder#630] [Q&A] [answered]
I am unable to install the lateset version of Subfinder using the command 
go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest

The above command executes successfully, but when I try to use Subfinder, the output is as 
Command 'subfinder' not found

**Comment by forgedhallpass** [comment, marked as answer]
The "installation" went successful, you just need to make sure to set up your environment correctly.
You need to add your `GOPATH/bin` to the `PATH` in your `.profile` (or `.zprofile`, depending on what shell are you using) configuration:

```sh
export GOPATH=$(go env GOPATH)
export PATH=$PATH:$GOPATH/bin
```
